### ISSUE TITLE
Running with system-icu USE flag enabled on a clean .eclipse folder (in home) Eclipse fails to start

### ISSUE THREAD

**Issue by stefan-langenmaier** [gentoo/eclipse-overlay#7]
When running with system-icu USE flag enabled on a clean .eclipse folder (in home) Eclipse fails to start.

It works if there is first an install with the USE flag system-icu disabled and Eclipse is started to create the .eclipse folder. Afterwards Eclipse can be reinstalled with the flag enabled and it will start. If the folder is now deleted the error from pull request #5 will show up again.

**Comment by hartwork** [comment]
The options seem to be:
- Produce a warning about requiring Oracle Java during emerge time and keep the `system-icu` flag
- Resolve flag `system-icu`, always unbundle ICU, require Oracle Java
- Resolve flag `system-icu`, never unbundle ICU

I'm unsure which road to go right now. What do you think?

**Comment by stefan-langenmaier** [comment]
I closed #6 because the problem is not related to oracle-jdk or icedtea. I have the same behaviour on both jdks. My issue report was wrong.

Concerning the USE flag I think the current behaviour is the best. Don't unbundle ICU by default, so this way it is working for everyone right from the start. But leave the other option available. I'd want it to be unbundled as much as possible. This is also why I would like to know if others can reproduce this issue.

I tried to compare the .eclipse folder (created by once by the bundled eclipse and once by the unbundled) but because the ICU error is so early it's not fully populated.
